### DISCUSSION TITLE
Serbo-Croatian feature suggestions

### DISCUSSION THREAD

**Discussion by SecularSteve** [WeblateOrg/weblate#10700] [Ideas] [answered]
Basically, these languages all encompass the same language under different regional dialects:
-Bosnian
-Serbian
-Croatian
-Montenegrin

Technically categorised under the following collective ISO-639-3 Code:
https://iso639-3.sil.org/code/hbs

My point is, the process to translate these languages can be a lot more streamlined if there was an easier way to transliterate between the dialects.

Instead of having to manually and tediously fill all 4 languages in with the same exact word/sentence and/or copy-paste them from the "Other Languages" page with little to no distinguishing differences used to modify them.

But that's just a suggestion.

**Comment by nijel** [comment, marked as answer]
Configure these as [secondary languages](https://docs.weblate.org/en/latest/user/profile.html#secondary-languages), and you will be able to copy them with a single click.

If there is a reliable way to automatically convert these, implementing this as a machine translation service would be possible.
